models/database.go

```diff
@@ -32,6 +32,9 @@ import (
 	"gopkg.in/reform.v1/dialects/postgresql"
 )
 
+// PMMPostgreSQLServiceName is a special Service Name representing PMM Server's PostgreSQL Service.
+const PMMPostgreSQLServiceName = "pmm-server-postgresql"
```

Comment:
I would call it `PMMServer…` to match `PMMServerAgentID` and `PMMServerNodeID`

Reply:
Renamed to `PMMServerPostgreSQLServiceName`, it's long but precise.